### DISCUSSION TITLE
My Block Requests

### DISCUSSION THREAD

**Discussion by rubiidev18alt** [cytrinc/bark-coding#79] [Block Requests]
## rubiidev18alt's requests
1. color and brightness effects from scratch (in a seperate looks category)
2. (not a block but a stylistic choice) no corners on blocks
3. previous costume
4. a few more blocks ported from scratch like glide to random position, a couple sensing blocks, and more

**Comment by nmsderp** [comment]
go to https://github.com/mariocraft987/bark.github.io/discussions/categories/block-requests

**Reply by rubiidev18alt** [reply]
this IS Block Requests.

**Reply by nmsderp** [reply]
sorry

**Comment by dumorando** [comment]
> (not a block but a stylistic choice) no corners on blocks

erm what the flip

**Reply by nmsderp** [reply]
erm what the sigma
